### DISCUSSION TITLE
supabase have too many personal access token. because of cli login

### DISCUSSION THREAD

**Discussion by matart15** [supabase/supabase#30610] [Questions]
I develop for multiple clients. they each have own supabase instance. 
That requires me to do `supabase login` and `supabase logout` everytime i switch.

I tried to invite single account to all of them but there is 2 free account limit
( initial development goes with free account. then later client decides which plan to buy )

supabase login create access token evertime. I end up with a lots of access token.

Is it intentional? or am i missing some workflow?

**Comment by silentworks** [comment]
Unless you are doing database interactions from your local computer there shouldn't be any need to be logging in and out of each client's instance. Each client should be using migration files so you can work on their project locally and then push to a GitHub repo which would have a workflow setup to automate running the migrations from GitHub actions. Manually doing these from your own machine isn't the best way when dealing with projects.

**Reply by matart15** [reply]
Thank you. For big projects we use Github actions to automatocally migrate.

But small projects it is too much. I just migrate changes manually. 
I understand it is not intended usage and we are not planning to write and maintain github action for small projects.

My concern is currently it is possible some users can use it like this.
is it going to be a problem if there is a lot of unused access tokens?
